src/internals/ir/callGraph.ts

```diff
@@ -38,14 +38,32 @@ class CGNode {
   public inEdges: Set<CGEdgeId> = new Set();
   public outEdges: Set<CGEdgeId> = new Set();
 
+  /**
+   * `astId` might be `undefined` if this node doesn’t have a corresponding AST entry,
```

Comment:
```suggestion
   * @param astId AST id of the relevant function definition. It might be `undefined` if this node doesn’t have a corresponding AST entry,
```